ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -222,7 +222,7 @@ StatusDialog {
 
         if(!!popup.preSelectedRecipient) {
             recipientLoader.selectedRecipientType = popup.preSelectedRecipientType
-            if (popup.preSelectedRecipientType == TabAddressSelectorView.Type.Address) {
+            if (popup.preSelectedRecipientType === TabAddressSelectorView.Type.Address) {
```

Comment:
I just removed this part of code in my PR so probably you could skip `SendModal` changes and we will not conflict!

Reply:
If you removed it, there will be no conflict